lib/screens/home_screen.dart

```diff
@@ -1,5 +1,4 @@
 import 'package:flutter/material.dart';
-import 'package:flutter/cupertino.dart';
 
 import '../api/pseudo_service.dart';
```

Comment:
import format, must be same as line 1 and 2

Reply:
noted sir. but i also found this from the dart github.

https://dart-lang.github.io/linter/lints/prefer_relative_imports.html

which method should i follow?

Reply:
There's nothing wrong with relative import. But as you will be working with existing projects and code, you would have to adjust depending on the standards of the project/team you are working with.

Reply:
okay sir. i understand. will be editing the files later. thanks!